microservice_install.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
this shouldn't exist here - if we don't have this handling provided in `deriv/perl` or `deriv/dzil`, please add it there instead